test/test_dataloader.py

```diff
@@ -62,6 +62,16 @@ def test_concat_raises_index_error(self):
             # this one goes to 11
             result[11]
 
+    def test_add_dataset(self):
+        d1 = TensorDataset(torch.rand(7, 3, 28, 28), torch.rand(7))
+        d2 = TensorDataset(torch.rand(7, 3, 28, 28), torch.rand(7))
+        d3 = TensorDataset(torch.rand(7, 3, 28, 28), torch.rand(7))
+        result = d1 + d2 + d3
+        self.assertEqual(21, len(result))
+        self.assertAlmostEqual(0, (d1[0][0] - result[0][0]).abs().sum())
```

Comment:
Shouldn't these be exactly equal to each other?

Reply:
thanks, I fixed it~